proxy/core/connection/connection.py

```diff
@@ -52,6 +52,7 @@ def connection(self) -> Union[ssl.SSLSocket, socket.socket]:
     def send(self, data: bytes) -> int:
         """Users must handle BrokenPipeError exceptions"""
         # logger.info(data)
+        logger.info('send to %s, data = |%s|', self.tag, data)
```

Comment:
remove or comment out?

Reply:
This log statement can be removed directly

Reply:
I didn't get you.  By having a `logger.info` in `send`, instance will keep printing all data sent to client and servers (even encrypted data).  So we should not have this enabled.  At-best we must have `.debug`, but even in debug mode, printing all raw data gets too verbose.

There is a reason I have a commented out line above.  I uncomment it when necessary for local debugging.